File: Restaurant/C++/Entree.cpp
#include "Entree.hpp"
#include <string>

Entree::Entree()
{}

Entree::~Entree()
{}

Entree::Entree(std::string name, double price)
{
	this->name = name;
	this->price = price;
}

std::string Entree::getName()
{
	return this->name;
}

double Entree::getPrice()
{
	return this->price;
}

File: Restaurant/Java/Entree.java
package restaurant;

class Entree extends Item {
	
	Entree(){
	}
	
	Entree(String name, double price)
	{
		this.name = name;
		this.price = price;
	}
	
	String getName() {
		return this.name;
	}
	
	double getPrice() {
		return this.price;
	}
	
}